.travis.yml

```diff
@@ -29,7 +29,7 @@ env:
 
 jobs:
   allow_failures:
-    - name: "Python 3.6 - PIP"
+    - name: "Python - PIP"
```

Comment:
Believe it or not, the PyPI-based one is passing! Allowing its failure for the future is probably good though.

Reply:
Yes that was my thought as well! Mostly because I want to minimize time spent debugging here in the future.